dom-event-delegation/main.ts

```diff
@@ -0,0 +1,15 @@
+const $task = document.querySelector('.task-list');
+
+if (!$task) throw new Error('.task-list down not exist');
+
+function handleClick(event: Event): void {
+  const eventTarget = event.target as HTMLElement;
+
+  if (eventTarget instanceof HTMLButtonElement) {
+    const checker = eventTarget.closest('.task-list-item');
+    console.log(checker);
```

Comment:
While this works, we were looking for you to use the `tagName` property of the `event.target`/`eventTarget`.

```
if ($eventTarget.tagName !== 'BUTTON') {
    return;
  }

  const $taskListItem = $eventTarget.closest('.task-list-item');

  console.log('closest .task-list-item', $taskListItem);
  $taskListItem?.remove();
  ```